cockatrice/src/dlg_settings.cpp

```diff
@@ -543,6 +510,91 @@ DeckEditorSettingsPage::DeckEditorSettingsPage()
     setLayout(lpMainLayout);
 }
 
+/**
+ * If reset, this method contains the default URLs we will populate
+ */
+void DeckEditorSettingsPage::resetDownloadedURLsButtonClicked()
+{
+    urlList->clear();
+
+    urlList->addItem("https://api.scryfall.com/cards/!uuid!?format=image");
+    urlList->addItem("https://api.scryfall.com/cards/multiverse/!cardid!?format=image");
+    // urlList->addItem("http://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=!cardid!&type=card");
```

Comment:
I think we should leave the gatherer urls in as lowest priority fallbacks.

In any case they shouldn't be committed commented out.